app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue

```
@@ -308,6 +329,7 @@ export default {
           v-if="telegramId"
           class="flex items-center gap-2 text-sm text-n-slate-11"
         >
+          <span class="text-n-slate-10">{{ $t('CONTACT_PANEL.LABEL_TELEGRAM_ID') }}</span>
```

ESLint

[error] Replace `·$t('CONTACT_PANEL.LABEL_TELEGRAM_ID')` with `⏎············$t('CONTACT_PANEL.LABEL_TELEGRAM_ID')⏎·········`

(prettier/prettier)